dashboard/pages/5_Word_Clouds.py

```diff
@@ -135,7 +149,29 @@ def run_app():
 
     st.title("Article Content Word Cloud by News Source")
 
-    fn_articles, dn_articles = get_all_article_content()
+    time_range_options = ["Last hour", "Last 24 hours", "Last 7 days"]
```

Comment:
good idea to use text to display the options rather than 1,24, 24*7 (which is less clear)